server/src/main/java/org/candlepin/resource/OwnerProductResource.java

```diff
@@ -196,7 +201,7 @@ protected Content fetchContent(Owner owner, String contentId) {
         response = Product.class, responseContainer = "list")
     @GET
     @Produces(MediaType.APPLICATION_JSON)
-    public CandlepinQuery<ProductDTO> listProducts(
+    public CandlepinQuery<ProductLegacyDTO> listProducts(
```

Comment:
We seem to be heavily using the "Legacy" product here and aren't using the base product at all, which implies it's not actually legacy.

Reply:
We are using both. ProductLegacyDTO and CandlepinAttributesDeserializer are my attempt to preserve current functionallity of @JsonSerialize/Deserialize we use for attributes.

Reply:
Removed *LegacyDTOs